lib.js

```diff
@@ -111,5 +109,7 @@ module.exports = (options) => {
                     }
                 });
             });
+        }).catch((error)=> {
+          console.error(`ERROR: ${error.detail || error.message || JSON.stringify(error)}`);
```

Comment:
We still want to reject the promise if there was an error, so we should not catch this error here.